packages/fast-components-react-msft/src/select/select.stories.tsx

```diff
@@ -154,12 +158,14 @@ storiesOf("Select", module)
             />
         </Select>
     ))
-    .add("Adjacent", () => (
+    .add("Right", () => (
         <Select
             placeholder="Select an option"
             onValueChange={action("onValueChange")}
             menuFlyoutConfig={{
                 horizontalPositioningMode: AxisPositioningMode.adjacent,
+                defaultHorizontalPosition: ViewportPositionerHorizontalPosition.right,
+                horizontalLockToDefault: true,
```

Comment:
This test seems to be using horizontal positioning adjacent and a default horizontal position right, since this is locked to default, is there any point in passing the `horizontalPositioningMode` and if not can you remove and add back another test for the adjact test?